client/src/components/Print/layouts/singleDay.tsx

```diff
@@ -0,0 +1,336 @@
+import React, { useMemo } from "react";
+import { PrintLayout } from "./types";
+import { Temporal } from "@js-temporal/polyfill";
+import {
+  useProgramDates,
+  useGroups,
+  usePkgs,
+  usePrograms,
+} from "../../TimetableV2/hooks";
+import { epochMillisecondsToPlainDateTime } from "../../TimetableV2/utils";
+import { ScheduledProgram } from "../../TimetableV2/types";
+
+interface SingleDayOptions {
+  date?: Temporal.PlainDate;
+  boldPkgs: (string | null)[];
+}
+
+type SingleDayValidOptions = SingleDayOptions;
+
+const timetable: PrintLayout<SingleDayOptions, SingleDayValidOptions> = {
+  label: "Na den",
+  initialOptions: {
+    date: undefined,
+    boldPkgs: [],
+  },
+  OptionsComponent: ({ options, setOptions }) => {
+    const days = useProgramDates();
+    const packages = usePkgs();
+    const day = options.date;
+
+    return (
+      <>
+        <div className="mb-3">
+          <label htmlFor="day">Den:</label>
+
+          <div className="d-flex flex-row gap-3 flex-wrap">
+            <div className="form-check">
+              <input
+                type="radio"
+                name="day"
+                id="allDays"
+                value="allDays"
+                className="form-check-input"
+                checked={!day}
+                onChange={() => {
+                  setOptions((prevOptions) => ({
+                    ...prevOptions,
+                    date: undefined,
+                  }));
+                }}
+              />
+              <label htmlFor="allDays" className="form-check-label">
+                Všechny dny
+              </label>
+            </div>
+
+            {days.map((dayOption) => (
+              <div key={dayOption.toString()} className="form-check">
+                <input
+                  type="radio"
+                  name="day"
+                  id={dayOption.toString()}
+                  value={dayOption.toString()}
+                  className="form-check-input"
+                  checked={day?.equals(dayOption)}
```

Comment:
I am getting these errors:
```
singleDay.tsx:58 A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://react.dev/link/controlled-components
```
It probably means that at the beginning it is undefined, this fix is working for me
```suggestion
                  checked={!!day?.equals(dayOption)}
```